code/style.css

```diff
@@ -19,12 +19,23 @@ h1 {
   color: white;
 }
 
+optgroup,
```

Comment:
i like the styling, the colors, the spacing and the images used! it looks really pretty!